axelrod/tests/unit/test_adaptive.py

```diff
@@ -0,0 +1,61 @@
+"""Test for the Adaptive strategy."""
+
+import axelrod
+
+from .test_player import TestHeadsUp, TestPlayer
+
+C, D = axelrod.Actions.C, axelrod.Actions.D
+
+
+class TestAdaptive(TestPlayer):
+
+    name = "Adaptive"
+    player = axelrod.Adaptive
+    expected_classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': False,
+        'makes_use_of': set(),
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def test_strategy(self):
+        # Test initial play sequence
+        self.responses_test([], [], [C] * 6 + [D] * 5)
+
+    def test_scoring(self):
+        player = axelrod.Adaptive()
+        opponent = axelrod.Cooperator()
+        player.play(opponent)
+        player.play(opponent)
+        self.assertEqual(3, player.scores[C])
+        game = axelrod.Game(-3, 10, 10, 10)
+        player.set_match_attributes(game=game)
```

Comment:
Hmmm... Yeah you're right that should do...  My comment was mainly motivated by the fact that the coverage report said it didn't hit those lines... Leave it as it is.